messages.js

```diff
@@ -77,8 +77,92 @@ MessageCreatorMorph.prototype = new DialogBoxMorph();
 MessageCreatorMorph.prototype.constructor = MessageCreatorMorph;
 MessageCreatorMorph.uber = DialogBoxMorph.prototype;
 
+
+
 function MessageCreatorMorph(target, action) {
+    this.minWidth = 0;
     this.init(target, action);
+    MessageCreatorMorph.prototype._fixLayout = DialogBoxMorph.prototype.fixLayout;
```

Comment:
Can you move this to the `init` function? Also, in this case, you can call the "super" class' fixLayout with `DialogBoxMorph.prototype.fixLayout.call(this);`